vck/src/commonTest/kotlin/at/asitplus/wallet/lib/data/rfc7515/CompactSerializedApplicationMediaTypeTest.kt

```diff
@@ -0,0 +1,54 @@
+package at.asitplus.wallet.lib.data.rfc7515
+
+import at.asitplus.wallet.lib.data.rfc7515.primitives.CompactApplicationMediaType
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.datatest.withData
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+
+class CompactSerializedApplicationMediaTypeTest : FreeSpec({
+    "t" - {
+        withData(
+            mapOf(
+                "application/jwt" to listOf(
+                    "application/jwt",
+                    "jwt",
+                    "application/jWt",
```

Comment:
Can we do something like 
```
input.map { if (kotlin.random.Random.nextBoolean()) it.uppercaseChar() else it.lowercaseChar() }.joinToString("")
```
to generate lower/uppercase input?

Reply:
removed